spec/factories/produce_item.rb

```diff
@@ -0,0 +1,8 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :produce_item do
+    name { Faker::Food.fruits }
```

Comment:
We have a unique constraint on `name`, so this could cause us trouble in the long run. To ensure uniqueness, we must use FactoryBot sequences:

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences

On a sidenote, I generally prefer using values that I know, so I'd opt for something like `banana` instead of `Faker::Food.fruits`. However, this is a matter of taste, so up to you deciding for Faker or a hardcoded value.